submitter/relayer/relayer.go

```diff
@@ -154,42 +159,39 @@ func (rl *Relayer) shouldResendCheckpoint(ckptInfo *types.CheckpointInfo, bumped
 // based on the current BTC load, considering both tx sizes
 // the result is multiplied by ResubmitFeeMultiplier set in config
 func (rl *Relayer) calculateBumpedFee(ckptInfo *types.CheckpointInfo) btcutil.Amount {
-	feeRate := rl.getFeeRate()
-	newTx1Fee := feeRate.FeeForVSize(ckptInfo.Tx1.Size)
-	newTx2Fee := feeRate.FeeForVSize(ckptInfo.Tx2.Size)
-	// minus the old fee of the first transaction because we do not want to pay again for the first transaction
-	bumpedFee := newTx1Fee + newTx2Fee - ckptInfo.Tx1.Fee
-
-	return bumpedFee.MulF64(float64(rl.config.ResubmitFeeMultiplier))
+	return ckptInfo.Tx2.Fee.MulF64(rl.config.ResubmitFeeMultiplier)
```

Comment:
why we don't want to estimate the fee here?

Reply:
We've changed how we estimate fee, by using the fee value from the response of `FundRawTransaction`. This produces a different value (a bit higher fee) than we used to, which means when we estimate the fee here it wouldn't be the same way as we did, and in local tests resulting in a lower calculated `bumpedFee` needing a higher `ResubmitFeeMultiplier`. So I see it either as estimating fee ourselves or using the response from `FundRawTransaction`. Open to any other inputs, I might be missing something

Reply:
This makes sense. Thanks for the explanation!